src/main/java/com/alphawallet/attestation/demo/SmartContract.java

```diff
@@ -29,7 +29,7 @@ public boolean verifyEqualityProof(byte[] com1, byte[] com2, ProofOfExponent pok
     return callFunction(function);
   }
 
-  public boolean testEncoding(FullProofOfExponent exp)
+  public boolean usageProofOfExponent(FullProofOfExponent exp)
   {
     Function function = checkEncoding(exp.getDerEncoding());
```

Comment:
The check encoding function seems to call the old function, instead of verifyAttestationRequestProofPayable. But in any case, it seems it is only called from Demo, which should still verifyEqualityProof

Reply:
Ah, I accidentally git overrode the fix to mapToCurveMultiplier which had the loop removed!